File: integrations/src/main/java/com/nivasafinance/services/voice/dto/VoiceCallResponse.java
package com.nivasafinance.services.voice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class VoiceCallResponse {
    private String callSid;
    private String status;
    private String fromNumber;
    private String toNumber;
    private Integer duration;
    private String recordingUrl;
    private String errorMessage;
}

File: integrations/src/main/kotlin/services/voice/dto/VoiceCallResponse.kt
package services.voice.dto

data class VoiceCallResponse(
    val callSid: String,
    val status: String,
    val fromNumber: String,
    val toNumber: String,
    val duration: Int? = null,
    val recordingUrl: String? = null,
    val errorMessage: String? = null
)
